AlphaWallet/Swap/Swap/ViewModels/SwapQuoteDetailsViewModel.swift

```diff
@@ -48,7 +48,7 @@ final class SwapQuoteDetailsViewModel {
                             type: step.type,
                             amount: pair.swapQuote.unsignedSwapTransaction.value,
                             amountUsd: pair.swapQuote.action.fromToken.priceUSD,
-                            estimate: pair.swapQuote.unsignedSwapTransaction.gasPrice ?? .zero,
+                            estimate: pair.swapQuote.unsignedSwapTransaction.gasPrice,
```

Comment:
I didn't test this, what would happen when the `gasPrice` is missing from the swap response with this change?

Maybe we should check with the li.fi folks too (but still good to have a fix ourselves)

Reply:
it just displays an error when fetching swap routes, i have no idea why it happens, something with api i suppose, because one request (eth-> usdt swap ) returns an error an the next same request returns sucessfull response.